src/utils/markdown-utils.ts

```diff
@@ -6,9 +6,9 @@ export enum Align {
 }
 
 export const Icon = {
-  skip: '⚪', // ':white_circle:'
-  success: '✅', // ':white_check_mark:'
-  fail: '❌' // ':x:'
+  skip: ':warning:',
+  success: ':white_check_mark:',
+  fail: ':x:'
 }
```

Comment:
This change causes incorrect rendering of icons in the result markdown.
The previous code was rendering icons in the code block.

We should keep the rendering as was.

Here is the diff:

![image](https://github.com/dorny/test-reporter/assets/287778/124a1ab9-2614-47b9-9b8b-8c87a994ec77)